docs/howtos/cut-a-new-release.md

```diff
@@ -1,39 +1,37 @@
-3. Cut the actual release.
+    **Note:** Because the release commit must always be on the main branch, your PR will need to be approved, CI successful, and merged before a release can be cut
+2. Cut the actual release.
     1. Click "Releases", and then "Draft a New Release" in the github UI.
     2. Enter `v<myversion>` as the tag. It's important this is the same as the tags that are in the changelog.
     3. Under the description, paste the contents of the release notes from CHANGELOG.md.
     4. Note that the release is not avaliable until the taskcluster build completes for that tag.
         - Finding this out takes a little navigation in the github UI. It's available at `https://github.com/mozilla/application-services/commits/v<VERSION NUMBER>` in the build status info (the emoji) next to the last commit.
         - If the taskcluster tag and/or release tasks fail, ping someone in slack and we'll figure out what to do.
-4. If you need to manually produce the iOS build for some reason (for example, if CircleCI cannot), someone with a mac needs to do the following steps:
-    1. If necessary, set up for performing iOS builds using `./libs/verify-ios-environment.sh`.
-    2. Run `./build-carthage.sh` in the root of the repository.
-    3. Upload the resulting `MozillaAppServices.framework.zip` as an attachment on the github release.
-5. In order for consumers to have access, we need to update in [android-components](https://github.com/mozilla-mobile/android-components).
+    5. Click "Publish Release".
+3. Inform consumers that the new release is available so that [android-components](https://github.com/mozilla-mobile/android-components) can be updated. If you will be creating the update, do the following:
     1. If the changes expose new functionality, or otherwise require changes to code or documentation in https://github.com/mozilla-mobile/android-components, perform those. This part is often done at the same time as the changes in application-services, to avoid being blocked on steps 3-4 of this document.
     2. Change the versions of our dependencies in [buildSrc/src/main/java/Dependencies.kt](https://github.com/mozilla-mobile/android-components/blob/master/buildSrc/src/main/java/Dependencies.kt).
     3. Note the relevant changes in their [docs/changelog.md](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md), and update the application-services version there as well in their list of dependency versions.
     4. **_Important: Manually test the changes versus the samples in android-components._**
         - We do not have automated test coverage for much of the network functionality at this point, so this is crucial.
-        - You can do this using the smoketest instructions in step (1) above.
-
+        - You can do this using the smoketest instructions below.
```

Comment:
as above :)

Reply:
Oh, I removed the smoke tests from step 1, and explicitly added them below.